.github/workflows/FirebaseDeploy.yml

```
@@ -0,0 +1 @@
+
```

actionlint (1.7.4)

workflow is empty

(syntax-check)